src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  return (
+    <>
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">{age && (`I am ${age}`)}</p>
```

Comment:
Better wrap the whole element into `age &&` condition, also don't write everything in one line, format it properly